install.sh

```diff
@@ -261,7 +348,22 @@ should_skip_install_into_existing_repositories() {
 #   0 in single repository install mode, 1 otherwise
 ############################################################
 is_single_repo_install() {
-    if [ "$SINGLE_REPO_INSTALL" = "yes" ]; then
+    if [ "$SINGLE_REPO_INSTALL" = "true" ]; then
+        return 0
+    else
+        return 1
+    fi
+}
+
+############################################################
+# Check if the install script is
+#   running in for a single repository without templates.
```

Comment:
This documentation (for `is_reset_all_standalone`) seems a copy-paste from `is_single_install`.

Reply:
Jup, leftover, remove it, thx!

Reply:
Not used!